libs/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,47 @@
+import { DateTimeRangeDto } from './datetime-range.dto';
+
+import { AnimeStatusDto } from './anime-status.dto';
+import { AnimeTypeDto } from './anime-type.dto';
+
+/** Anime DTO. */
+export type AnimeDto = {
+
+	/** ID. */
+	readonly id: number;
+
+	/** Created date. */
+	readonly created: string;
+
+	/** Modified date. */
+	readonly modified: string;
+
+	/** English title. */
+	readonly title_eng: string;
+
+	/** Japanese title. */
+	readonly title_jpn: string;
+
+	/** Image URL. */
+	readonly image: string;
+
+	/** Aired duration. */
+	readonly aired: DateTimeRangeDto;
+
+	/** Type. */
+	readonly type: AnimeTypeDto;
+
+	/** Status. */
+	readonly status: AnimeStatusDto;
+
+	/** Score. */
+	readonly score: number | null;
+
+	/** User score. */
+	readonly user_score: number | null;
+
+	/** List of studios. */
+	readonly studios: readonly number[];
+
+	/** List of genres. */
+	readonly genres: readonly number[];
+};
```

Comment:
Try this, it will reduce code when type get too long and enhance readability
```suggestion
export type AnimeDto = Readonly<{

	/** ID. */
	id: number;

	/** Created date. */
	created: string;

	/** Modified date. */
	modified: string;

	/** English title. */
	title_eng: string;

	/** Japanese title. */
	title_jpn: string;

	/** Image URL. */
	image: string;

	/** Aired duration. */
	aired: DateTimeRangeDto;

	/** Type. */
	type: AnimeTypeDto;

	/** Status. */
	status: AnimeStatusDto;

	/** Score. */
	score: number | null;

	/** User score. */
	user_score: number | null;

	/** List of studios. */
	studios: readonly number[];

	/** List of genres. */
	genres: readonly number[];
}>;
```